Bundle/Bundle-NonWGSScenario5-ProductsofConception-Example.json

```diff
@@ -0,0 +1,922 @@
+    ]
+   },
+   "response": {
+    "status": "200 OK",
+    "outcome": {
+     "resourceType": "OperationOutcome",
+     "issue": [
+      {
+       "severity": "information",
+       "code": "informational",
+       "diagnostics": "No issues detected during validation"
+      }
+     ]
+    }
+   }
+  },
+  {
+   "fullUrl": "urn:uuid:c60f7e06-79b1-4a34-9979-e8a461b3f7c8",
+   "resource": {
+    "resourceType": "Patient",
+    "id": "c60f7e06-79b1-4a34-9979-e8a461b3f7c8",
+    "extension": [
+     {
+      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-BirthSex",
+      "valueCodeableConcept": {
+       "coding": [
+        {
+         "system": "http://terminology.hl7.org/CodeSystem/v3-AdministrativeGender",
+         "code": "F",
+         "display": "Female"
+        }
+       ]
+      }
+     },
+     {
+      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-EthnicCategory",
+      "valueCodeableConcept": {
+       "coding": [
+        {
+         "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-EthnicCategoryEngland",
+         "code": "A",
+         "display": "White - British"
+        }
+       ]
+      }
+     }
+    ],
+    "identifier": [
+     {
+      "extension": [
+       {
+        "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-NHSNumberVerificationStatus",
+        "valueCodeableConcept": {
+         "coding": [
+          {
+           "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-NHSNumberVerificationStatusEngland",
+           "version": "1.1.1",
+           "code": "01",
+           "display": "Number present and verified"
+          }
+         ]
+        }
+       }
+      ],
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449308322"
+     },
+     {
+      "system": "urn:oid:2.16.840.1.113883.2.1.3.2.4.18.24",
+      "value": "71636034",
+      "assigner": {
+       "identifier": {
+        "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+        "value": "RAX01"
+       }
+      }
+     }
+    ],
+    "name": [
+     {
+      "given": [
+       "Ryanne"
+      ],
+      "family": "Boulder"
+     }
+    ],
+    "link": [
+     {
+      "other": {
+       "reference": "https://int.api.service.nhs.uk/personal-demographics/FHIR/R4/Patient/9449308322"
+      },
+      "type": "seealso"
+     }
+    ]
+   },
+   "response": {
+    "status": "200 OK",
+    "outcome": {
+     "resourceType": "OperationOutcome",
+     "issue": [
+      {
+       "severity": "information",
+       "code": "informational",
+       "diagnostics": "No issues detected during validation"
+      }
+     ]
+    }
+   }
+  },
+  {
+   "fullUrl": "urn:uuid:037d8ff3-da35-4fad-9d28-7a120d4fffe9",
+   "resource": {
+    "resourceType": "Patient",
+    "id": "037d8ff3-da35-4fad-9d28-7a120d4fffe9",
+    "extension": [
+     {
+      "url": "http://hl7.org/fhir/6.0/StructureDefinition/extension-Patient.fetalStatus",
+      "valueCode": "fetal-demise"
```

Comment:
For fetal demise, i would also expect a deceasedDate and possibly an observation recording the pregnancy outcome, e.g. Stillbirth

Reply:
Observation: Past pregnancy history of stillbirth(urn:uuid:3e58d52e-aff3-4a70-956b-28509c664f59) currently exist.